profiles/common/modules/features/multisite_twitterblock/multisite_twitterblock.module

```diff
@@ -199,15 +198,16 @@ function multisite_twitterblock_theme() {
 
 /**
  * Implements theme hook.
- * @global bool $is_https
  *
  * @param array $variables
- *   array of new variables
+ *   Array of new variables.
+ *
+ * @global bool $is_https
  *
  * @return string
- *   twitter output config
+ *   Twitter output config.
  */
-function theme_fpfis_twitter($variables) {
+function theme_fpfis_twitter(array $variables = array()) {
```

Comment:
Don't add ` = array()` unless the array is optional. The code expects that there is a `$variables['type']` set, so it is not optional in this case.